pages/people/[id]/case-note.tsx

```diff
@@ -151,7 +155,8 @@ export const getServerSideProps: GetServerSideProps = async ({
   params,
   query,
 }) => {
-  let submission;
+  let submission: Submission;
+
   if (query.submissionId) {
     submission = await getSubmissionById(String(query.submissionId));
```

Comment:
we should just assert that query.submissionId is a string, we don't need to actually call the String constructor which is adding real runtime code.

`query.submissionId as string`